spec/factories/automobiles.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :automobile do
+    brand { "MyString" }
+    model { "MyString" }
+    year { "" }
```

Comment:
Year and Price are integers, not strings

Reply:
Done